src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/CaCase.kt

```diff
@@ -0,0 +1,62 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.CaViewCasesTab
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceKind
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import java.time.LocalDate
+
+@Schema(description = "Describes an CA(OMU) case")
+data class CaCase(
+
+  @Schema(description = "Type of this licence", example = LicenceKinds.CRD)
+  val kind: LicenceKind? = null,
+
+  @Schema(description = "Unique identifier for this licence within the service", example = "99999")
+  val licenceId: Long? = null,
+
+  @Schema(description = "The version number of this licence", example = "1.3")
+  val licenceVersionOf: Long? = null,
+
+  @Schema(description = "The full name of the person on licence", example = "John Doe")
+  val name: String,
+
+  @Schema(description = "The prison identifier for the person on this licence", example = "A9999AA")
+  val prisonerNumber: String,
+
+  @Schema(description = "The details for the active supervising probation officer")
+  val probationPractitioner: ProbationPractitioner? = null,
+
+  @Schema(description = "The date on which the prisoner leaves custody", example = "30/11/2022")
+  @JsonFormat(pattern = "dd/MM/yyyy")
+  val releaseDate: LocalDate?,
```

Comment:
**thought: Passing back release date as LocalDate format instead of string and we can handle date not logic in backend.**